apps/avifdec.c

```diff
@@ -79,7 +79,11 @@ int main(int argc, char * argv[])
     while (argIndex < argc) {
         const char * arg = argv[argIndex];
 
-        if (!strcmp(arg, "-h") || !strcmp(arg, "--help")) {
+        if (!strcmp(arg, "--")) {
+            // Strop parsing flags, everything after this is positional arguments
```

Comment:
Strop => Stop

Since the `syntax()` function refers to these as "options" rather "flags", perhaps this comment should also say "options"?

Also in avifenc.c.